docs/modules/ROOT/pages/cops.adoc

```diff
@@ -97,6 +97,7 @@
 === Department xref:cops_rspec_factorybot.adoc[RSpec/FactoryBot]
 
 * xref:cops_rspec_factorybot.adoc#rspecfactorybot/attributedefinedstatically[RSpec/FactoryBot/AttributeDefinedStatically]
+* xref:cops_rspec_factorybot.adoc#rspecfactorybot/consistentparenthesesstyle[RSpec/FactoryBot/ConsistentParenthesesStyle]
```

Comment:
Unrelated to your changes. Anchors for cops in sub-departments (Capybara, FactoryBot) seem broken. Worth looking at in another PR, and probably the problem is in RuboCop repo.

Should be:
```diff
- #rspecfactorybot/consistentparenthesesstyle
+ #rspecfactorybotconsistentparenthesesstyle
```